.github/workflows/build-php-db-import-export.yml

```diff
@@ -121,6 +119,12 @@ jobs:
         name: Checkout
         uses: actions/checkout@v3
 
+      - name: Configure AWS Credentials
```

Comment:
U IE libky nakoniec musim robit cez tieto credentials len build. @Halama na bucket nemozem mat docasne cred pretoze to nepodporuju dalsie backendy ktore tu mame takze som pre to musel zachovat staticke cred. Tie cred ale nemaju pristup nikam inam len na ten bucket..

Reply:
ok, to pak chce vyresit v tech backendech. treba pro SNFLK je to spatne tam se posilaji primo staticke credentials...